index.js

```diff
@@ -8,6 +8,8 @@ var mkdirp     = require('mkdirp')
 var rimraf     = require('rimraf')
 var mdm        = require('mdmanifest')
 var fs         = require('fs')
+var valid      = require('muxrpc-validation')
+valid.set(require('./validators'))
```

Comment:
this is still global for `muxrpc-validation` because it mutates data in that module. this could cause problems if there is more than one muxrpc-validation dep in the tree (which one are you referencing?)
what would be better would be to have mrv return a setup function.
`var valid = require('muxrpc-validation')(require('./validators'))`

Although, at that point, you should just do:
``` js
//validators.js
module.exports = require('muxrpc-validation')({
...
})
```
and in index.js
``` js
var valid = require('./validators')
```
then the validators file entirely sets up and describes the validator functions.